From 0d7e4ae8a39ce1982619d77ec9f03e437d84ba50 Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Thu, 24 Jan 2013 15:35:12 +0100
Subject: [PATCH 086/184] am33xx: disable NAPI

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 arch/arm/boot/dts/am33xx.dtsi |    1 +
 1 file changed, 1 insertion(+)

"""
!!! TODO ? disable-napi is not referenced in kernel-3.15 or kernel-3.17 or am335x SRM
do we need this feature?
diff --git a/arch/arm/boot/dts/am33xx.dtsi b/arch/arm/boot/dts/am33xx.dtsi
index 912cfcd..4b20654 100644
--- a/arch/arm/boot/dts/am33xx.dtsi
+++ b/arch/arm/boot/dts/am33xx.dtsi
@@ -525,6 +525,7 @@
 			 */
 			interrupts = <40 41 42 43>;
 			ranges;
+			disable-napi;
 
 			davinci_mdio: mdio@4a101000 {
 				compatible = "ti,davinci_mdio";
"""
-- 
1.7.9.5

